packages/sanity/src/desk/panes/document/inspectors/changes/index.ts

```diff
@@ -0,0 +1,18 @@
+import {RestoreIcon} from '@sanity/icons'
+import {lazy} from 'react'
+import {DocumentInspector} from 'sanity'
+
+export const changesInspector: DocumentInspector = {
+  name: 'changes',
```

Comment:
Could it be good to have a more unique name for this inspector, so that we minimize the risk of someone creating an inspector with the name that interferes with this inspector?  Perhaps a name like `desk/changes` would be a good name, so you understand that it is an inspector that `sanity/desk` configures in e.g. the `document.inspectors` resolver?

Reply:
Yes, true. Changing to `sanity/desk/history`